docs/Basics-Dimensions.md

```diff
@@ -0,0 +1,63 @@
+---
+id: basics-dimensions
+title: Dimensions
+layout: docs
+category: The Basics
+permalink: docs/basics-dimensions.html
+next: basics-layout
+---
+
+A component's dimensions determine its size on the screen.
+
+#### Fixed Dimensions
+
+The simplest way to set the dimensions of a component is by adding a fixed `width` and `height` to style. All dimensions in React Native are unitless, and represent density-independent pixels.
+
+```ReactNativeWebPlayer
+import React from 'react';
+import { AppRegistry, View } from 'react-native';
+
+class AwesomeProject {
```

Comment:
Yeah, seems to run without `Component`. Seems weird. Since we are calling `registerComponent` on something that isn't technically a `Component`. Is there some sort of implied extension when we omit it or something?